src/cli/eest/commands/clean.py

```diff
@@ -11,18 +9,20 @@
 
 @click.command(short_help="Remove all generated files and directories.")
 @click.option(
-    "--all", is_flag=True, help="Remove the virtual environment and .tox directory as well."
+    "--all",
+    "all_files",
+    is_flag=True,
+    help="Remove the virtual environment and .tox directory as well.",
 )
 @click.option("--dry-run", is_flag=True, help="Simulate the cleanup without removing files.")
 @click.option("-v", "--verbose", is_flag=True, help="Show verbose output.")
-def clean(all: bool, dry_run: bool, verbose: bool):
+def clean(all_files: bool, dry_run: bool, verbose: bool):
```

Comment:
Some changes are specific to variable names. `all` is a standard python variable name hence the change to `all_files`. You will see similar changes for example the `input` var name cannot be used too.